repository/BaselineOfSmalltalkCI.package/BaselineOfSmalltalkCI.class/instance/pharoVersionsFrom..st

```diff
@@ -0,0 +1,4 @@
+baselines
+pharoVersionsFrom: aVersion
+
+	^ (aVersion to: SystemVersion current major) collect: [ :v | ('pharo' , v asString , '.x') asSymbol ]
```

Comment:
Turns out this only works in Pharo:
```
a MessageNotUnderstood occurred (error 2010), a Set does not understand  #'current'
```

Reply:
Works in Squeak 5.1 as well. Is that error from GS?

Reply:
I don't know tbh.

Reply:
Sure, but do we really need to complicate this even more? I forgot about the semantics, but maybe the spec could just target `#'pharo'` instead (or `#'pharoCommon'`?) and then we have specific overrides for Pharo <  5 (e.g. `#'pharo4.x'`)?